docs/overview.rst

```diff
@@ -0,0 +1,39 @@
+.. _overview_label:
+
+~~~~~~~~
+Overview
+~~~~~~~~
+
+`oemoflex` provides a framework to perform a set of energy system scenarios with `oemof-solph <https://oemof-solph.readthedocs.io/>`_.
+
+
+Use cases
+=========
+
+In oemoflex, scenarios are called **use cases**.
```

Comment:
That's true for the first part of the experiment.

Reply:
I would propose to change this: Let us call them scenarios. The scenarios in the first part of the projec Flexme are no full scenarios, but in the second part they will be.